app/connectors/index.js

```diff
@@ -1,9 +1,6 @@
-export { default as accountsPage } from "./accountsPage";
 export { default as purchaseTickets } from "./purchaseTickets";
 export { default as send } from "./send";
 export { default as ticketsList } from "./ticketsList";
 export { default as ticketsOverview } from "./ticketsOverview";
 export { default as routing } from "./routing";
-export { default as modalVisible } from "./modalVisible";
-export { default as network } from "./network";
 export { default as lnPage } from "./lnPage";
```

Comment:
I'm getting an error on startup.  BlurableContainer in components/layout is still referring to modalVisible.

Reply:
Yep, rebase smoke, will let you know when all good.

Reply:
Fixed @alexlyp